runtime/lua/vim/lsp/semantic_tokens.lua

```diff
@@ -331,6 +331,10 @@ function STHighlighter:process_response(response, client, version)
     tokens = response.data
```

Comment:
Probably be better to change this line instead to `tokens = response.data or {}`?